src/tests/unit/recipeService/getRecipesByTitle.test.ts

```diff
@@ -0,0 +1,46 @@
+import RecipeService from '../../../services/recipeService';
+import { Recipe } from '../../../models/recipeModel';
+import {
+  mockTitle,
+  mockBody,
+  mockImage,
+  saveRecipe,
+  mockId,
+} from '../../mocks/mockRecipe';
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('getRecipesByTitle', () => {
+  test('should fetch recipes from database', async () => {
+    await new Recipe({
+      ownerId: mockId,
+      title: mockTitle,
+      imageUrl: mockImage,
+      body: mockBody,
+    }).save();
+    const recipes = await recipeService.getRecipesByTitle(mockTitle);
+    expect(recipes[0]).toBeDefined();
+    expect(recipes[0].title).toBe(mockTitle);
+    expect(recipes[0].imageUrl).toBe(mockImage);
+    expect(recipes[0].body).toBe(mockBody);
+  });
+
+  test('should find recipes with titles containing a query', async () => {
+    const recipe = await saveRecipe();
+    const partTitle = recipe.title.split(' ')[2];
```

Comment:
This assumes the title has at least 2 spaces in the name. I'd rather use something like first or last X characters.

```ts
const secondHalf = title.substring(title.length - title.length / 2)
```

Reply:
Titles generated for these tests all have 3 words. I don't see much difference between this and your proposition.

Reply:
If this is constant behaviour, I'm ok with it.